.stickler.yml

```diff
@@ -0,0 +1,3 @@
+linters:
+  eslint:
+    config: './eslint.config'
```

Comment:
Change this path like this 👇 
```yaml
linters:
  eslint:
    config: './.eslintrc.json'
```

Reply:
- I followed your suggestion and I still have 3 non-code related errors. Should I change the linting style from **Airnb** to **standard** ?